param/cobalt.yaml

```diff
@@ -160,3 +160,8 @@ sensor:
             x: 0
             y: 0
             z: 0
+hydrophones:
+    positions:
+        x: 3
+        y: 3
+        z: 3
```

Comment:
Are these positions representing distance from the origin? They could be distance from the reference, therefore making the reference position (0,0,0) as in derivation.